src/client/App/AppRoutes.tsx

```diff
@@ -18,6 +19,7 @@ const App = (props: AppProps) => {
 			<Route path="/patterns" element={<Patterns />}></Route>
 			<Route path="/patterns/:id" element={<PatternDetails />}></Route>
 			<Route path="/patterns/new" element={<AddPattern />}></Route>
+			<Route path="/patterns/admin" element={<AdminPanel />}></Route>
```

Comment:
Would it maybe make more sense to have /admin be its own route instead of a sub-route of patterns?

Reply:
I'm going to do a full site admin later, this was a requirement that I intend to delete when that other site admin is done. It's a specific admin page only for patterns